src/d3a/models/area/markets.py

```diff
@@ -68,7 +69,7 @@ def _market_rotation(self, current_time, markets, past_markets, area_agent):
                 else:
                     first = False
                 self.log.trace("Moving {t:%H:%M} {m} to past"
-                               .format(t=timeframe, m=past_markets[timeframe].area.name))
+                               .format(t=timeframe, m="past_markets[timeframe].area.name"))
```

Comment:
You can circumvent the "" by using past_markets[timeframe].name (I assume the name of the market is the same as the name of the area, please correct me if my assumption is wrong).

Reply:
yes, sorry. This was temporary. I forgot to change this. Will do thanks!